lib/facter/realm.rb

```diff
@@ -1,32 +1,31 @@
-require 'facter/util/sssd'
-
-if defined? Facter::Util::Sssd
+require_relative 'util/realm'
+if defined? Facter::Util::Realm
   # == Fact: foreman_ipa
   Facter.add(:foreman_ipa, :type => :aggregate) do
     {
       :default_realm => 'global/realm',
       :default_server => 'global/server',
     }.each do |key, path|
       chunk(key) do
-        val = Facter::Util::Sssd.ipa_value(path)
+        val = Facter::Util::Realm.ipa_value(path)
         {key => val} if val
+        # == Fact: foreman_sssd
       end
     end
   end
-
   # == Fact: foreman_sssd
   Facter.add(:foreman_sssd, :type => :aggregate) do
     {
       :services => 'target[.="sssd"]/services',
+      :server => 'target[.=~regexp("domain/.*")][1]/ipa_server',
```

Comment:
Since this is a list, should it be `servers`?

Reply:
It is an array to should at least have one member. Since the original configuration option is `ipa_server`, I went with `server`. I think it is relevant if it is either one.

Reply:
I still think it should be `servers` here since it's an array. The name should give a hint about the data type to expect.

Reply:
OK, done.